setup.py

```diff
@@ -13,7 +13,7 @@
   download_url = 'https://github.com/bodyshopbidsdotcom/githubgateway/tarball/%s' % current_version,
   keywords = ['api', 'gateway', 'http', 'REST'],
   install_requires = [
-    'basegateway==0.10'
```

Comment:
Why `0.12` instead of `0.11`?

Reply:
because of this https://github.com/bodyshopbidsdotcom/basegateway/blob/master/setup.py#L3